### PR TITLE
@next Icon: add ri-flag

### PR DESCRIPTION
**Pull request by music1353**

### CODE REVIEW

File: src/@next/Icon/icons/ri-flag-line.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 18 19" xmlns="http://www.w3.org/2000/svg">
```
Comment (ninariccimarie) [inline src/@next/Icon/icons/ri-flag-line.svg]: @music1353 can you double check the exported file? it seems the size is a bit different 🤔

Comment (music1353) [reply src/@next/Icon/icons/ri-flag-line.svg]: @ninariccimarie thanks adjusted!

Comment (ninariccimarie) [reply src/@next/Icon/icons/ri-flag-line.svg]: @music1353 it should probably be the other way around. the size is usually the one with the 0 0 24 24 instead. with the hollow space around the icon.

File: src/@next/Icon/icons/ri-flag-fill.svg
Comment (ninariccimarie) [inline src/@next/Icon/icons/ri-flag-fill.svg:1]: Screenshot 2024-06-03 at 4 23 07 PM
@music1353 please check for this one. it seems the lines were missed during the export?

**Review by ninariccimarie — APPROVED** [review]
LGTM!

**Pull request merged**